client/state/selectors/test/is-app-banner-visible.js

```diff
@@ -0,0 +1,77 @@
+import { expect } from 'chai';
+import { isAppBannerVisible } from 'calypso/state/selectors/is-app-banner-visible';
+
+describe( 'isAppBannerVisible()', () => {
+	test( 'should return false if Terms or Service update banner is displayed', () => {
```

Comment:
> Terms or Service

typo :D

but maybe just ToS would suffice?

Reply:
Ops 😄 
In latest refactor, `isAppBannerVisible` became `shouldDisplayAppBanner` so to not overlap with `isAppBannerVisible` and `isAppBannerDismissed` from state.ui